docs/workflows/copy.rst

```diff
@@ -47,47 +47,57 @@ depend on the existence of other content units to be "correct" or meaningful. Fo
+
+.. note::
 
-The goal is to be as easy to use as possible while maintaining the "correctness" of the repository.
+    In all such cases, Pulp3 engages in a "best effort" attempt to fulfill the requirement. If
+    the source repositor(ies) don't contain all of the referenced content, the copy will proceed
+    without them.
+
+The goal is to be as easy to use as possible, while allowing for a "best effort' at maintaining
+the "correctness" of the repository **when requested to do so**.
 
 Dependency solving
 __________________
 
-When copying RPM packages between repositories, it is likely desirable to ensure that all of the
-RPM dependencies they need to be "installable" on a client system are also present in the
-repository they are being copied to. For example, if you want to add the "hexchat" RPM to a
-new repository, and you want to be able to install it from that repository, the repository should
-also contain its dependencies such as "libnotify" and "gtk3". This applies likewise to all RPM
-concepts that "contain" RPMs also, such as Advisories (Errata), Modules, and PackageGroups. With
-the RPM copy API, you are afforded the option to have all dependencies (and the dependencies of
-those dependencies) copied for you automatically in the background, if they do not already exist
-in the destination repository. In Pulp 2, this feature was called "recursive copy" because in a
-sense it copies dependencies recursively.
+When copying RPM packages, advisories, or modules between repositories, it is useful to ensure
+that all of the RPM dependencies they need to be "installable" on a client system are also present
+in the destination-repository.
+
+For example: if you want to add the "hexchat" RPM to a new repository, and you want to be able
+to install it from that repository, the repository should also contain hexchat's dependencies
+such as "libnotify" and "gtk3".
+
+This applies to all RPM concepts that "contain" RPMs, such as Advisories (Errata), Modules, and
+PackageGroups. With the RPM copy API, you are afforded the option to have all dependencies (and
+the dependencies of those dependencies) copied for you automatically, if they do not already exist
+in the destination repository. In Pulp 2, this feature was called "recursive copy".
 
 Unfortunately, solving these complex dependency relationships can be quite expensive. For this
-reason, the functionality to perform these checks is not enabled by default, and is instead
+reason, the functionality to perform these checks is **not enabled by default**, and is instead
 provided by setting the "dependency_solving" parameter to a value of True when making calls against
-the API. Note that if you do elect to not perform dependency solving, it will be possible to
-accidentally create incomplete repositories, if you are not careful.
+the API. Note that if you do not choose dependency-solving, it is possible to create incomplete
+repositories.
```

Comment:
Also possible to create incomplete repositories if the configuration is incorrect, if packages aren't provided to the solver then "best-effort" won't be good enough.